plugins/automation/admin/assets/javascripts/admin/components/automation-list.gjs

```diff
@@ -211,9 +213,10 @@ export default class AutomationList extends Component {
                   </LinkTo>
 
                   <DButton
+                    @isLoading={{this.isDestroying}}
```

Comment:
@SamSaffron Sorry, it wasn't clear from my suggestion but the `this.isDestroying` property doesn't exist and needs to be implemented.

Reply:
I implemented it in my followup commit, it is a very very very hard to repro edge case, but it is handled now.

not supporting a second click after the first click to delete was a bigger issue.